src/agents/customer_group.jl

```diff
@@ -1443,7 +1429,7 @@ function solve_agent_problem!(
         customers.d(h, t, :) .= customers.d_my(reg_year_index, h, t)
     end
 
-    if hem_opts isa HEMOptions{VIU, NullUseCase, SupplyChoice, <:UseCase}
+    if hem_opts isa HEMOptions{VIU, NullUseCase, NullUseCase, <:UseCase}
         WholesaleMarketPerc = 0.01
     else
         WholesaleMarketPerc = 1.0
```

Comment:
The revised logic is incorrect. Are we keeping the SupplyChoice use case (at least momentarily) or not?